docs/architecture/adr-007-specialization-groups.md

```diff
@@ -0,0 +1,179 @@
+# ADR 007: Specialization Groups
+
+## Changelog
+
+- 2019 Jul 31: Initial Draft
+
+## Context
+
+This idea was first conceived of in order to fulfill the use case of the
+creation of a decentralized Computer Emergency Response Team (dCERT), Whose
+members would be elected by the Cosmos Community and would fulfill the role of
```

Comment:
What is the "Cosmos Community"? Are you referring specifically to the Atom holder set?